### DISCUSSION TITLE
RFM22 testing

### DISCUSSION THREAD

**Discussion by lw2dtz** [jgromes/RadioLib#314] [Q&A]
Hello,

I am testing a RFM22 module and an 8mhz arduino with the last library version, in the following modes, in RTTY and Hellschreiber it has worked as indicated. In morse it transmits but instead of having a silence between brand and brand it remains like a carrier.
Any idea what can it be?

Regards.

Gustavo.

**Comment by jgromes** [comment]
Addressed in #313
